CHANGES.md

```diff
@@ -4,8 +4,10 @@ Change Log
 ### 1.30 - 2017-02-01
 * Deprecated
     * The properties `url` and `key` will be removed from `GeocoderViewModel` in 1.31. These properties will be available on geocoder services that support them, like `BingMapsGeocoderService`.
+    * The function `createBinormalAndBitangent` of `GeometryPipeline` will be removed in 1.31. Use the function `createTangentAndBitangent` instead. [#4856](https://github.com/AnalyticalGraphicsInc/cesium/pull/4856)
 * Breaking changes
     * Removed separate `heading`, `pitch`, `roll` parameters from `Transform.headingPitchRollToFixedFrame` and `Transform.headingPitchRollQuaternion`. Pass a `headingPitchRoll` object instead. [#4843](https://github.com/AnalyticalGraphicsInc/cesium/pull/4843)
+    * The property `binornmal` has been renamed to `bitangent` throughout. [#4856](https://github.com/AnalyticalGraphicsInc/cesium/pull/4856)
```

Comment:
Be more precise. `Geometry`, `VertexFormat`, etc.  Otherwise, it isn't obvious to users where this occurs.